lrrbot/stream_status.py

```diff
@@ -33,6 +34,12 @@ def start_check_stream(self):
 	async def check_stream(self):
 		log.debug("Checking stream")
 		data = await twitch.get_info(use_fallback=False)
+		if self.old_title != data['title']:
+			log.debug("Title changed, parsing for show information")
+			self.old_title = data['title']
+			if self.find_show(self, data['title']):
```

Comment:
`find_show` is not on `self` but `self.lrrbot`. Also `self` gets passed automatically.
```suggestion
			if self.lrrbot.find_show(data['title']):
```